### DISCUSSION TITLE
Repository structure for master is not compliant

### DISCUSSION THREAD

**Discussion by neTix84** [hass-emulated-hue/core#241] [Q&A]
Hi, 
I have the following problem.
I try to add the repository to HACS in Home Assistant Core.
When I enter the link to the repository (https://github.com/hass-emulated-hue/hassio-repo), I get the following message: "Repository structure for master is not compliant"

Any idea what I do wrong or is the repository incompatible with HACS?

Thanks Danny.

Edit: I found this (https://hacs.xyz/docs/faq/custom_repositories)
WARNING: Not all repositories will work in HACS, since HACS still needs the repository to have a known structure.
Would it be possible to create a HACS compatible repro?

System:

Version | 2021.1.0
-- | --
Installation Type | Home Assistant Container
Development | false
Supervisor | false
Docker | true
Virtual Environment | false
Python Version | 3.8.7
Operating System Family | Linux
Operating System Version | 5.10.1-Unraid
CPU Architecture | x86_64
Timezone | Europe/Berlin

**Comment by alexyao2015** [comment]
This is not a HACS component. You'll need to add it to the addon store instead. This is in the supervisor tab.

**Reply by neTix84** [reply]
Is it possible to make available or convert it to HACS component? Or totally different?

I have no Supervisor tab in Home-Assistant-Core in Docker Container :(

**Reply by alexyao2015** [reply]
Either run a supervised installation of home assistant or run it manually as in #78
